### DISCUSSION TITLE
apply new conditions type to matchers-condition / condition

### DISCUSSION THREAD

**Discussion by wen0750** [projectdiscovery/nuclei-templates#6974] [Ideas]
There are two conditions type provided in nuclei. But in some cases, XOR(exclusive OR) "Maybe" can reduce my workload, such as checking a plugin version. Since I am green on Nuclei so this may bad idea.

**Comment by ehsandeep** [comment]
@wen0750 do you have an example of a matcher or condition you are looking to write?

**Comment by olearycrew** [comment]
@wen0750 when you can, please provide the additional information we were looking for. Otherwise we'll soon consider this discussion closed...thanks!
